packages/assets-controllers/src/MultichainBalancesController/MultichainBalancesController.ts

```diff
@@ -139,6 +145,8 @@ export class MultichainBalancesController extends BaseController<
   MultichainBalancesControllerState,
   MultichainBalancesControllerMessenger
 > {
+  #isUnlocked = true;
```

Comment:
As discussed together, we should check on the `KeyringController` state to get the initial value. We cannot assume it's unlocked by default.

We can use something like this:
- https://github.com/MetaMask/core/blob/main/packages/assets-controllers/src/TokenDetectionController.ts#L331-L334